automatic/apache-httpd/tools/chocolateyBeforeModify.ps1

```diff
@@ -0,0 +1,7 @@
+$configFile = Join-Path $env:chocolateyPackageFolder 'config.json'
```

Comment:
I'm guessing this should actually be a config.json file located in the tools directory, not the root package directory.